### PR TITLE
docs: dan v2 overview

### PR DESCRIPTION
**Pull request by CjS77**
- deprecates old DAN RFCs
 - Adds introductory RFC, describing DANv2 at a high level
 - Includes an RFC for the turbine model

### CODE REVIEW

**Review by SWvheerden — APPROVED** [review]
ACK

File: src/RFC-0303_DanOverview.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+The aim of this Request for Comment (RFC) is to describe the key elements of the Tari second layer, also known as the 
+Digital Assets Network (DAN).
+
+## Related Requests for Comment
+
+* [RFC-0100: Base Layer](RFC-0100_BaseLayer.md)
+
+## Description
+
+The Tari DAN utilises a sharded BFT consensus mechanism called [Cerberus](https://arxiv.org/abs/2008.04450).
```
Comment (stringhandler) [inline src/RFC-0303_DanOverview.md:60]: There may be some misinterpretations about which parts of Cerberus are used
```suggestion
The Tari DAN is based on a sharded BFT consensus mechanism called [Cerberus](https://arxiv.org/abs/2008.04450). 
```

Comment (stringhandler) [reply src/RFC-0303_DanOverview.md:60]: Of particular note is that the paper uses pBFT (3 rounds) whereas Tari currently uses Hotstuff (4 rounds)

File: src/RFC-0303_DanOverview.md [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+instruction that it aids in getting finalised.
+
+### DAN consensus layer
+
+The Cerberus BFT consensus algorithm runs on the consensus layer. This layer is completely ignorant of the 
+semantics of DAN smart contracts.
+
+This layer only cares that:
+* _only_ VNs that have registered on the base layer are participating in consensus.
+* VNs are self-organising into VN committees and are carrying out the rules of Cerberus and emergent HotStuff
```
Comment (stringhandler) [inline src/RFC-0303_DanOverview.md:149]: Emergent Hotstuff has not be defined or reference anywhere. I think its inclusion here is unneeded
```suggestion
* VNs are self-organising into VN committees and are carrying out the rules of Cerberus
```

File: src/RFC-0303_DanOverview.md [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+The Cerberus BFT consensus algorithm runs on the consensus layer. This layer is completely ignorant of the 
+semantics of DAN smart contracts.
+
+This layer only cares that:
+* _only_ VNs that have registered on the base layer are participating in consensus.
+* VNs are self-organising into VN committees and are carrying out the rules of Cerberus and emergent HotStuff 
+  correctly.
+
+In particular, the consensus layer has _no idea_ whether an instruction's output is correct. If two-thirds (plus one)
+of the committee say that the results are correct, then consensus has been reached and the consensus layer is happy.
```
Comment (stringhandler) [inline src/RFC-0303_DanOverview.md:153]:
```suggestion
of the committee agree on the results, then consensus has been reached and the consensus layer is happy.
```

Comment (stringhandler) [reply src/RFC-0303_DanOverview.md:153]: It's perhaps also worth mentioning that the result they agree on could be an error in executing the code.

File: src/RFC-0303_DanOverview.md [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+
+There is no peg out back to the base layer for Thaums. Thaum holders wishing to convert back to Tari will be able to 
+perform a submarine swap with a Tari seller. We also anticipate that exchanges will list the Thaum-Tari pair to 
+enable easy conversion of Thaums back to Tari.
+
+# Change Log
+
+| Date        | Change              | Author     |
+|:------------|:--------------------|:-----------|
+| 1 Nov 2022  | High-level overview | CjS77      |
```
Comment (stringhandler) [inline src/RFC-0303_DanOverview.md:182]: Some RFCs have this in ascending order, some in descending. Not sure what the standard should be

File: src/RFC-0320_TurbineModel.md [hunk 5]
```diff
@@ -0,0 +1,175 @@
...
+
+This document and its content are intended for information purposes only and may be subject to change or update
+without notice.
+
+This document may include preliminary concepts that may or may not be in the process of being developed by the Tari
+community. The release of this document is intended solely for review and discussion by the community of the
+technological merits of the potential system outlined herein.
+
+## Goals
+
```
Comment (stringhandler) [inline src/RFC-0320_TurbineModel.md:50]:
```suggestion
## Definitions
```

File: src/RFC-0320_TurbineModel.md [hunk 6]
```diff
@@ -0,0 +1,175 @@
...
+## Description
+
+Side-chains are related to their parent chains via a pegging mechanism. In general, peg-in transactions (transferring
+value to the side-chain) are straightforward. One locks value up on the parent chain, which can be referenced by the
+side-chains. However, the reverse transaction is fraught with difficulty, since the parent chain must know almost
+nothing about the transaction particulars of the side chain. We know this is the case, otherwise the entire
+side-chain + parent-chain system are forced to work in lock-step and the two chains are really just one larger, more 
+complicated chain.
+
+Peg outs are particularly difficult if the participants change on the side-chain (as opposed to say, payment
```
Comment (stringhandler) [inline src/RFC-0320_TurbineModel.md:70]:
```suggestion
Peg outs are also particularly difficult if the participants change on the side-chain (as opposed to say, payment 
```

**Review by stringhandler — DISMISSED** [review]
Happy. Some optional suggestions

File: src/RFC-0303_DanOverview.md (hunk 4 above)
Comment (CjS77) [reply src/RFC-0303_DanOverview.md:182]: I also noticed the lack of consistency and decided to go with it for now :)

**Comment by CjS77** [comment]
ACK

**Pull request merged**